pimd/pim_assert.c

```diff
@@ -65,10 +65,10 @@ void pim_ifassert_winner_set(struct pim_ifchannel *ch,
 		if (winner_changed) {
 			char was_str[INET_ADDRSTRLEN];
 			char winner_str[INET_ADDRSTRLEN];
-			pim_inet4_dump("<was?>", ch->ifassert_winner, was_str,
```

Comment:
NAK, this function should be removed and addresses printed with `printfrr` directly (`%pI4`/`%pI6` previously, for pimd a new `%pPA` would be best.)

Reply:
Hi David, this change is done because we are going to add separate apis for v4 and v6. Please refer https://github.com/FRRouting/frr/pull/10194. We are trying to use pim_str.h for pimv4 compilation and pim6_str.h for pimv6 compilation.

Reply:
Yes I agree David on this. We need to add similar to %pI4 and %pI6. We will take it up in separate PR since that is a different set of work. Hence have removed this change.